lib/samples/show_service_area/show_service_area_sample.dart

```diff
@@ -25,53 +25,107 @@ class ShowServiceAreaSample extends StatefulWidget {
 }
 
 class _ShowServiceAreaSampleState extends State<ShowServiceAreaSample> {
-  final _serviceAreaTask = ServiceAreaTask.withUrl(Uri.parse(
-      'https://route-api.arcgis.com/arcgis/rest/services/World/ServiceAreas/NAServer/ServiceArea_World'));
-
+  // create a map view controller
   final _mapViewController = ArcGISMapView.createController();
-  final _facilityPointGraphicsOverlay = GraphicsOverlay();
-  final _serviceAreaGraphicsOverlay = GraphicsOverlay();
-  bool _isInitialized = false;
-  late final ServiceAreaParameters _serviceAreaParameters;
 
-  final _facilityPointSymbol = SimpleMarkerSymbol(
-    color: Colors.white,
-    style: SimpleMarkerSymbolStyle.circle,
-    size: 10,
-  )..outline = SimpleLineSymbol(
-      style: SimpleLineSymbolStyle.solid,
-      color: Colors.black,
-      width: 2,
+  // create graphics overlays for displaying facilities and barriers and apply renderers for the symbology.
+  final _facilityGraphicsOverlay = GraphicsOverlay()
+    ..renderer = SimpleRenderer(
+      symbol: PictureMarkerSymbol.withUrl(Uri.parse(
+          'https://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png')),
+    );
+  final _barrierGraphicsOverlay = GraphicsOverlay()
+    ..renderer = SimpleRenderer(
+      symbol: SimpleFillSymbol(
+          style: SimpleFillSymbolStyle.diagonalCross, color: Colors.red),
     );
 
-  final _serviceAreaSymbol = SimpleFillSymbol(
-    style: SimpleFillSymbolStyle.solid,
-    color: const Color(0x40FF3232),
-    outline: SimpleLineSymbol(
-      style: SimpleLineSymbolStyle.solid,
-      color: Colors.black,
-      width: 1,
+  // create a graphics overlay for displays service area results and a list of symbols applied for each impedence cutoff added to the service area parameters.
+  final _serviceAreaGraphicsOverlay = GraphicsOverlay();
+  final _serviceAreaSymbols = [
+    SimpleFillSymbol(
+      color: Colors.yellowAccent.withOpacity(0.4),
+      outline: SimpleLineSymbol(color: Colors.yellow.withOpacity(0.8)),
     ),
-  );
+    SimpleFillSymbol(
+      color: Colors.orangeAccent.withOpacity(0.4),
+      outline: SimpleLineSymbol(color: Colors.orange.withOpacity(0.8)),
+    ),
+    SimpleFillSymbol(
+      color: Colors.greenAccent.withOpacity(0.4),
+      outline: SimpleLineSymbol(color: Colors.green.withOpacity(0.8)),
+    ),
+  ];
+
+  // create a service area task used to find service areas around a facility.
+  final _serviceAreaTask = ServiceAreaTask.withUrl(Uri.parse(
+      'https://route-api.arcgis.com/arcgis/rest/services/World/ServiceAreas/NAServer/ServiceArea_World'));
+  late final ServiceAreaParameters _serviceAreaParameters;
+
+  bool _ready = false;
+  Selection _segmentedButtonSelection = Selection.facility;
```

Comment:
Well here's an interesting one. This could be:
```dart
var _ready = false;
var _segmentedButtonSelection = Selection.facility;
```
For some reason we've been using `bool _ready = false;` when our standard would be to use `var`. There's probably no reason to make an exception for `bool`.